plugins/repository-azure/src/main/java/org/elasticsearch/cloud/azure/blobstore/AzureBlobContainer.java

```diff
@@ -83,6 +85,19 @@ public InputStream openInput(String blobName) throws IOException {
     }
 
     @Override
+    public void writeBlob(String blobName, InputStream inputStream, long blobSize) throws IOException {
+        try (OutputStream stream = createOutput(blobName)) {
```

Comment:
Agreed. In Azure SDK I have a simple upload(InputStream stream, long length)  method which is I believe obvious to use here.
Do you think I should do it within the same PR or send a new one after this one?